`dist/cardano-addresses.d.ts` has broken references on `src/*.ts` files

# How to reproduce

Compile ts source code with `tsx`

# Expected behavior

Compiled successfully

# Actual behavior

The compilation error:

```
node_modules/cardano-addresses/dist/cardano-addresses.d.ts:1:22 - error TS6053: File '/home/alexey/my-project/node_modules/cardano-addresses/dist/src/foreign.d.ts' not found.

1 /// <reference path="src/foreign.d.ts" />
                       ~~~~~~~~~~~~~~~~

node_modules/cardano-addresses/dist/cardano-addresses.d.ts:8:37 - error TS2307: Cannot find module './foreign' or its corresponding type declarations.

8 import { CardanoAddressesApi } from './foreign';
                                      ~~~~~~~~~~~
```

# How to fix

Fixing paths fixing the compilation error:

```ts
/// <reference path="../src/foreign.d.ts" />
/**
 * Typescript bindings for `Cardano.Address`.
 *
 * @module
 */
import { Address, XPub, InspectAddress } from './types';
import { CardanoAddressesApi } from '../src/foreign';
```

--- a/jsapi/src/cardano-addresses.ts
+++ b/jsapi/src/cardano-addresses.ts
@@ -1,4 +1,4 @@
-/// <reference path="./foreign.d.ts" />
+/// <reference path="../src/foreign.d.ts" />
 
 // Copyright © 2021 IOHK
 // License: Apache-2.0
@@ -10,7 +10,7 @@
  */
 
 import { Address, XPub, InspectAddress } from './types';
-import { CardanoAddressesJSModule, CardanoAddressesApi } from './foreign';
+import { CardanoAddressesJSModule, CardanoAddressesApi } from '../src/foreign';
 
 /**
  * Get information about a Cardano address. Three address formats are

